### ISSUE TITLE
Quick questions

### ISSUE THREAD

**Issue by robwolff3** [geekofweek/homeassistant#10]
Hey Geekofweek,

Long time follower of your repo. Just gave it a once over to see new changes and additions. Three questions I have if you don't mind answering when it convenient:

- Is there any benefit to using the ecobee sensor for occupancy and temperature as appose to a z-wave multi sensor? Is it unnecessarily complex to use home assistant for HVAC logic as appose to ecobee?
- It seems that the room occupancy detection for triggering lights would have to go though ecobees cloud before HA as appose to being a local call through your network. Is there any noticeable delay using this method? (I assume this creates a dependency on the internet connection.)
- What are your plans for presence detection now that Locative is EOL? Do you have a second best option?

Thanks!

**Comment by geekofweek** [comment]
- Not really.  I used them because Ecobee was really my first smart home product prior to even getting into HA, so I had them, and they worked.  They're not super great for turning a room occupied for on the fly automations, but work well as conditions or if a room goes unoccupied.  I do some HVAC logic with HA, but for the most part Ecobee does it's normal thing with room occupancy.  Where I deviate is setting it in comfort profiles for different actions.  Example:  If I turn on my fireplace, I adjust the profile to one that ignores both the living room and dining room as they are closer to the fireplace.  That prevents the rest of the house from getting cold being overpowered by the fireplace and Ecobee not running the central heat.  I also automate around Home and Away, I find it faster if I do it instead of letting Ecobee do it.  By default Ecobee does it all, I augment with some overrides here in there in HA.

- See above, they are not great at triggering on occupancy.  I do for some, but for lights it isn't great.  It really isn't the cloud API that is an issue at all.  _Side rant, I find the whole cloud API argument a non issue for me 99% of the time.  If my internet goes down, I am way more irritated at no internet than the automations not working.  I get the if they go out of business argument, or they have an outage, fair arguments.  In reality hasn't been as big of a deal to me.  If they offer local I go that route, if not I don't loose sleep over it._ Ecobee sensors are slow to trigger occupancy by the nature of what they are designed to do, they're not meant to be used in that way but to manage an HVAC system which handles some delay.  I find it to be anywhere from 1-3 mins.

- This is a tough one, for now Locative is still the most reliable option I've tried.  Providing the API password option doesn't go away and iOS doesn't break it in a future release, it still has some life left.  I've been testing the native iOS app reliability with locations and will probably move that direction eventually.  The other option I've mulled over is, if I can find the time, forking the locative app and bringing it up to snuff with the newer HA auth.  I just haven't had enough time to dedicate to such a project yet.
